storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst

```diff
@@ -35,28 +35,33 @@ Setting up a slave for replication with |Percona XtraBackup| is really a very st
 * ``Percona XtraBackup``
   The backup tool we will use. It should be installed in both computers for convenience.
 
+.. note:: It is not recommended to mix MySQL variants
+   (Percona Server, MySQL, MariaDB) in your replication setup.
+   This may produce incorrect :file:`xtrabackup_slave_info` file
+   when adding a new slave.
+
 STEP 1: Make a backup on ``TheMaster`` and prepare it
 =====================================================
 
 At ``TheMaster``, issue the following to a shell:
 
 .. code-block:: console
 
-   TheMaster$ innobackupex --user=yourDBuser --password=MaGiCdB1 /path/to/backupdir 
+   TheMaster$ xtrabackup --user=yourDBuser --password=MaGiCdB1 /path/to/backupdir 
```

Comment:
This command will become `xtrabackup --backup --user=yourDBuser --password=MaGiCdB1 --target-dir=/path/to/backupdir`